### DISCUSSION TITLE
Creating Teams

### DISCUSSION THREAD

**Discussion by merkourisa** [merkourisa/Interaction-Design-Project-Assignment#2] [Announcements]
**To create your team (2-3 fellow students) follow the above steps:** _Deadline:18/03/2022_

1. After you have created your Github account make a new comment by inserting the above table in your comment. In this way, you can create a new team and invite new members. 
2. If you would like to join a team you can ask to join by making the appropriate comment. If there is no team available you must create a new one.
3. Every time there is a change in the team (new member) you have to update the teams details by editing the comment, so it's always visible which teams are active and which are their members.

Name of the team: XXX
GitHub url: [XXX](url)
Accepting new member: Yes/No

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | dpsdXXXXX | gh link |
| 2 | dpsdYYYYY | gh link |
| 3 | ... | ... | 

This discussion is for organizing the teams. For any other question, you can open a new discussion.

**Comment by Jelizog** [comment]
Name of the team: GENIUS MONKEY 
Accepting new members: Νο
Administrator: [THANOS](https://github.com/ThanosSl)

| # | ΑΜ | GitHub profile | Approved |
| -- | -- | -- | -- |
| 1 | dpsd19120 | [ Profile](https://github.com/ThanosSl) | OK |
| 2 | dpsd19115 | [ Profile](https://github.com/Tsavvis) | OK |
| 3 | dpsd19034 | [Profile](https://github.com/Jelizog) | OK |

**Reply by merkourisa** [reply]
@Ebabouraki you should create a new comment for your team, not  reply on @Jelizog comment. The comment for your team should have a similar structure of @Jelizog comment to be approved. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by Jelizog** [comment]
> 

Στην περιοχή του GitHub URL τι πρέπει να βάλουμε;
In the area of GitHub URL, what's the correct information that must be inserted?

**Reply by merkourisa** [reply]
@Jelizog if you are the organizer of your team you should insert the link of your [forked repository](https://github.com/Jelizog/Interaction-Design-Project-Assignment)

**Reply by Jelizog** [reply]
Can you check If I've done it correctly?

**Reply by merkourisa** [reply]
@Jelizog perfect

**Comment by DPSD19140** [comment]


**Reply by Stathenia** [reply]
> !

> Name of the team: GENIUS MONKEY GitHub URL: XXX Accepting new member: Νο
> 
> #	ΑΜ	GitHub profile
> 1	dpsd19140	[ Profile](https://github.com/DPSD19140)
> 2	dpsd19014	[ Profiles](https://github.com/ZoiGalanidpsd19014)
> 3	dpsd19103      [ Profiles](https://github.com/Stathenia)

**Reply by merkourisa** [reply]
@Stathenia see my [comment ](https://github.com/merkourisa/Interaction-Design-Project-Assignment/discussions/2#discussioncomment-2304152)and make the correct one for your team

**Comment by itsdjdannysy** [comment]
> 

I'm about to create a new team!

**Reply by dpsd18057** [reply]
count me in

**Comment by papadimitriou19093** [comment]


**Reply by merkourisa** [reply]
@papadimitriou19093  you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by Costas-Lemonis** [comment]
Name of the team: **SOLID TEAM**
GitHub URL: [URL](https://github.com/Costas-Lemonis/Interaction-Design-Project-Assignment) 
Accepting new member: Νο

| #  | ΑΜ | GitHub profile | Approved |
| ------------- | ------------- | ------------- |  ------------- |
| 1  | dpsd18132  | [Profile](https://github.com/Chatzilyras-Michalis)  | OK |
| 2  | dpsd19065  | [Profiles](https://github.com/Costas-Lemonis)  | OK |
| 3  | dpsd19055  | [Profiles](https://github.com/Giannis-Kokkinis)  | OK |

**Reply by merkourisa** [reply]
@Costas-Lemonis you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by dpsd19061** [comment]
Name of the team: STRAGGLEAS (╯°□°）╯︵ ┻━┻
GitHub URL: <a href="https://github.com/dpsd19061/Interaction-Design-Project-Assignment">URL</a>
Accepting new member: Νο
|#| AM  | GitHub profile | Approved |
| -- | -- | -- | -- |
|1| dpsd19061  | [Profile](https://github.com/dpsd19061)  | OK |
|2| dpsd19080  | [Profile](https://github.com/dpsd19080)  | OK |
|3| dpsd19125  | [Profile](https://github.com/MichailAngelosSotirchos)  | OK |

**Reply by merkourisa** [reply]
@dpsd19061 you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Reply by dpsd19061** [reply]
@merkourisa I made some changes. Can you please check if everything is done correctly?

**Reply by merkourisa** [reply]
@dpsd19061  perfect

**Comment by MariaSiafali117** [comment]
Name of the team: Upcoming Designers
GitHub url: [URL](https://github.com/MariaSiafali117/Interaction-Design-Project-Assignment) 
Accepting new member: No


| # |     ΑΜ                | GitHub profile  | Approved |
|---| --------------     |----------------  |--------------|
|1|   dpsd19117   |	[Profile](https://github.com/MariaSiafali117)  | OK |
|2|  dpsd19118   |  [Profile](https://github.com/StellaSiafali)           | OK | 
|3|   dpsd19074   |  [Profile](https://github.com/AnnaMitsiani)       | OK |

**Reply by merkourisa** [reply]
@MariaSiafali117 you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by papadimitriou19093** [comment]


**Reply by merkourisa** [reply]
@papadimitriou19093 you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by dpsd16047** [comment]
Name of the team: Auga
GitHub url: [Url](https://github.com/dpsd16047/Interaction-Design-Project-Assignment)
Accepting new member: No


| # | AM | GitHub Profile | Approved | 
| :---         |     :---:      |      ---: | -- | 
| 1   | dpsd16015     | [Profile](https://github.com/dpsd16015)    | OK | 
| 2   | dpsd16047     | [Profile](https://github.com/dpsd16047)    | OK | 
| 3   | dpsd16121     | [Profile](https://github.com/dpsd16121)    | OK |

**Comment by dpsd18057** [comment]
Name of the team: Apithanoi
GitHub url: [URL](https://github.com/dpsd18057/Interaction-Design-Project-Assignment)
Accepting new member: No

| # | ΑΜ | GitHub profile | Approved | 
| -- | -- | -- | -- | 
| 1 | dpsd18057 | [profile](https://github.com/dpsd18057) | OK | 
| 2 | dpsd18134 | [profile](https://github.com/xatzzz) | OK | 
| 3 | dpsd19124 | [profile](https://github.com/itsdjdannysy) | OK |

**Reply by itsdjdannysy** [reply]
le goooooo

**Comment by MaritettaKatsarou** [comment]
Name of the team: Partners In Crime
GitHub [URL](https://github.com/MaritettaKatsarou/Interaction-Design-Project-Assignment)
Accepting new member: Νο

| # | AM | GitHub Profiles |  Approved | 
| :---      |     :---:      |          ---: | -- | 
| 1   | dpsd19051     | [MaritettaKatsarou](https://github.com/MaritettaKatsarou)   | ΟΚ | 
| 2   | dpsd19070    | [ArtemisMalliakoudi](https://github.com/Artem1sM)   | ΟΚ | 
| 3    | dpsd19090     | [VaggelisPanagiaris](https://github.com/VaggelisPanagiaris) |  ΟΚ |

**Reply by Artem1sM** [reply]
ψαχνω ομαδα αν δεν εχετε βρειι

**Reply by merkourisa** [reply]
@Artem1sM CodeBenders are accepting members. Have a look [here](https://github.com/merkourisa/Interaction-Design-Project-Assignment/discussions/2#discussioncomment-2340056)

**Comment by Ebabouraki** [comment]
Name of the team: SMURFS
GitHub URL: [URL](https://github.com/Ebabouraki/Interaction-Design-Project-Assignment)
Accepting new member: Νο

| # | ΑΜ | GitHub profile | Approved |
| -- | -- | -- | -- |
| 1 | dpsd19009 | [ Profile](https://github.com/Ebabouraki) | ΟΚ | 
| 2 | dpsd19005 | [ Profile](https://github.com/Marina-Eleni-Anastasiadou) | ΟΚ | 
| 3 | dpsd19011 | [Profile](https://github.com/DPSD19011) | ΟΚ |

**Reply by merkourisa** [reply]
@Ebabouraki correct

**Comment by FaidraDamianou** [comment]
Name of the team: CodeBenders *:･ﾟ✧*:･ﾟ✧
GitHub URL : [URL](https://github.com/FaidraDamianou/Interaction-Design-Project-Assignment)
Accepting new member: No
| # |       AM         |    Names     | Approved | 
| -- | -------------  | ------------- | -- | 
| 1 | dpsd18059    |[Profile](https://github.com/MARINAKYRIAKOU)| OK | 
| 2 | dpsd19028  | [Profile](https://github.com/FaidraDamianou)| OK |
| 3 | dpsd18108 | [Profile](https://github.com/dpsd18108) | OK |

**Reply by NFCcloud** [reply]
Can I enter you team ?

**Reply by FaidraDamianou** [reply]
@dpsd18108 done ψυχούλα

**Comment by CharisisLalopoulos** [comment]
Name of the team: **ハデハデダ✧** ( HadeHadeDa )
GitHub url: [URL](https://github.com/CharisisLalopoulos/Interaction-Design-Project-Assignment)
Accepting new member: No

|**#** | **AM** | **GitHub profile** | **Approved** |
| ----- | -------- | -------------------- | --------------- |
| 1 | dpsd19030 | [EvelinaDespotidou](https://github.com/Evedes01) | OK |
| 2 | dpsd19040 | [sotoszogos](https://github.com/sotoszogos) | OK |
| 3 | dpsd19064 | [CharisisLalopoulos](https://github.com/CharisisLalopoulos) | OK |

**Comment by ZoiGalanidpsd19014** [comment]
Name of the team: Team Rocket
GitHub url: [URL](https://github.com/DPSD19140/Interaction-Design-Project-Assignment)
Accepting new member: No

|**#** | **AM** | **GitHub profile** | **Approved** |
| ----- | -------- | -------------------- | -- |
| 1 | dpsd19103 | [StatheniaParaskevopoulou](https://github.com/Stathenia) | ok |
| 2 | dpsd19140 | [SofiaPantelopoulou](https://github.com/DPSD19140) | ok  |
| 3 | dpsd19014 | [ZoiGalani](https://github.com/ZoiGalanidpsd19014) | ok  |

**Comment by AggelosTzia** [comment]
Name of the team: LubenGitHub 
URL: [URL](https://github.com/AggelosTzia/Interaction-Design-Project-Assignment) 
Accepting new member: Νο

 
 | #   | AM                |    GitHub profile                                          | Approved |
 | :-- | --                   | ----:                                                             | -- |
 |1    |  dpsd19128   |   [Profile](https://github.com/AggelosTzia)| ok |
 |2    |  dpsd19113   |   [Profile](https://github.com/dpsd19113)  | ok |
 |3    |  dpsd19142   |   [Profile](https://github.com/dpsd19142)  | ok |

**Reply by dpsd19113** [reply]
tsapou

**Reply by merkourisa** [reply]
@AggelosTzia and @dpsd19113 you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables). 

Please delete any inappropriate messages above, this is an official discussion.

**Comment by BYKORDOU** [comment]
Name of the team: BIRTHCAKE
GitHub URL: [URL](https://github.com/RaniaKrz/Interaction-Design-Project-Assignment)
Accepting new member: No

| # | AM  | Profile | Approved |
|-----------| ------------- | ------------- | ------------- |
| 1 | dpsd19138  | [Αγγελική Κορδού](https://github.com/BYKORDOU) | ok |
| 2 | dpsd19003 | [Αναστασία Άμπα](https://github.com/AnastasiaAmp) | ok  |
| 3 |dpsd19047|[Ράνια Καρούζου](https://github.com/RaniaKrz) | ok |

**Reply by merkourisa** [reply]
@BYKORDOU you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by dpsd19097** [comment]
Name of the team: Dark Side
GitHub URL: [URL](https://github.com/dpsd19097/Interaction-Design-Project-Assignment)
Accepting new member: Yes

| # | ΑΜ | GitHub profile | Approved | 
| -- | -- | -- | -- | 
| 1 | dpsd19121 | [ Profile](https://github.com/mar-stamp) | OK | 
| 2 | dpsd19097 | [ Profile](https://github.com/dpsd19097) | OK |

**Reply by Sophiapapadopoulou** [reply]
Μπορω να μπω στην ομαδα;

**Comment by DChristos** [comment]
Team Name : NoName
GitHub URL : [URL](https://github.com/DChristos/Interaction-Design-Project-Assignment)
Accepting new members : No 
| #| AM | GitHub Profile | Approved | 
| --- | --- | --- | -- | 
| 1 | dpsd17084 | [Profile 1](https://github.com/GeorgePachis) |  OK | 
| 2 | dpsd14074 |[Profile 2](https://github.com/DChristos) |  OK | 
| 3 | dpsd14035 |[Profile 3](https://github.com/dpsd14035) |  OK |

**Comment by KostisG** [comment]


**Reply by merkourisa** [reply]
@KostisG  you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by dpsd19001** [comment]
Name of the team: GEOCHR
GitHub URL:  [URL](https://github.com/dpsd19001/Interaction-Design-Project-Assignment)
| # | AM | Github Profile | Approved   | 
| :---         |     :---:      |          ---: | --  | 
|1   | dpsd19001   |[PROFILE](https://github.com/dpsd19001)  | ok  | 
| 2   | dpsd19016  |[PROFILE](https://github.com/dpsd19016)  | ok  |

**Comment by StellaPiperaki** [comment]
Name of the team: SE
GitHub URL: [URL](https://github.com/StellaPiperaki/Interaction-Design-Project-Assignment)
Accepting new members  :yes
| # | AM | Github Profile | Approved |
| :---         |     :---:      |       ---: |  ------: |
|1   | dpsd19109   |[PROFILE](https://github.com/StellaPiperaki)  | ok  | 
| 2   | dpsd19036  |[PROFILE](https://github.com/dpsd19036)  |  ok  | 
| 3   |   |  |

**Comment by FoiviPetropoulou** [comment]
Name of the team: PB&F
GitHub URL: [URL](https://github.com/FoiviPetropoulou/Interaction-Design-Project-Assignment)
Accepting new member: Νο

| # | ΑΜ | GitHub profile | Approved |
| -- | -- | -- | -- |
| 1 | dpsd19108 | [ Petropoulou](https://github.com/FoiviPetropoulou) |  ok  | 
| 2 | dpsd19072 | [ Mauridis](https://github.com/panosmavridis) |  ok  | 
| 3 | dpsd19092 | [ Papavasileiou](https://github.com/dpsd19092) | ok  |

**Comment by sdoulaveris** [comment]
Name of the team: agio pneuma
GitHub URL: [URL](https://github.com/sdoulaveris/Interaction-Design-Project-Assignment)
Accepting new member: ΟΧΙ
| # | AM | PROFILE |  | 
| :---         |     :---:      |          ---: | --  | 
| 1   | dpsd19033 | [Spiros ](https://github.com/sdoulaveris)   | ok  | 
| 2    | dpsd19041       | [Lydia ](https://github.com/lydiaznt)   | ok  | 
| 3   | dpsd19013       | [Tony](https://github.com/tonygavridakis)    | ok  |

**Comment by athenast** [comment]
Name of the team: perikokladoules
GitHub URL:  [URL](https://github.com/athenast/Interaction-Design-Project-Assignment)

| # | AM | Github Profile | Approved  | 
| ---         |     ---:      |          ---: |  -- | 
| 1 | dpsd19122| [Athena](https://github.com/athenast)     | ok |      
| 2 | dpsd19032| [Eva](https://github.com/evadost)            | ok |       
| 3 | dpsd19044| [Dimitris](https://github.com/dpsd19044)| ok |

**Comment by Artem1sM** [comment]
Name of the team: 
GitHub URL: [URL](https://github.com/Artem1sM/Interaction-Design-Project-Assignment)
Accepting new member: Yes

| # | ΑΜ | GitHub profile | Approved |
| -- | -- | -- | -- |
| 1 | dpsd19070 | [ Artem1sM](https://github.com/Artem1sM) | 
| 2 | dpsd | [ ](https://github.com/) | 
| 3 | dpsd | [ ](https://github.com/) |

**Comment by Kuletsi** [comment]
Name of the team: asterakia
GitHub URL: [URL](https://github.com/Kuletsi/Interaction-Design-Project-Assignment)

|  #  | AM            |  Profile                                              | Approved | 
| --- |------------| --------------------------------------- | -- | 
|  1  |dpsd19139 |[Gelena Ilektra](https://github.com/Kuletsi)  | ok | 
|  2  |dpsd19015 |[Christina](https://github.com/cgalani) | ok | 
|  3  |dpsd19143 |[Evita](https://github.com/EvitaXen) | ok |

**Reply by merkourisa** [reply]
@Kuletsi please add Evita's link

**Comment by DespoinaKanni** [comment]
Name of the team: strike
GitHub url: [URL](https://github.com/DespoinaKanni/Interaction-Design-Project-Assignment) 
Accepting new member: Yes


| # |     ΑΜ                | GitHub profile  | Approved |
|--| --------------     |----------------  |--------------|
|1|   dpsd17042   |	[Profile](https://github.com/DespoinaKanni)  | ok  |
|2|  dpsd17111   |  [Profile](https://github.com/fotistsiampas)      | ok |

**Comment by panostzannet** [comment]
| # | AM  | Profile | Approved |
|-----------| ------------- | ------------- | ------------- |
| 1 | dpsd19127 | [Panagiotis Tzannetakos](https://github.com/panostzannet) |  |
| 2 | dpsd19105 | [Dimitris Pavliotis](https://github.com/dpavliotis) |   |
| 3 | dpsd19143 | [Christina Kallontzi](https://github.com/dpsd19043) |  |

**Reply by merkourisa** [reply]
@panostzannet you should display the information about your team in a way similar to the one used by @Jelizog. Here you can see some help on how to [organize information with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by IoannaNi** [comment]
Name of the team: Super Pink
GitHub URL: [URL](https://github.com/IoannaNi/Interaction-Design-Project-Assignment)
Accepting new member: Νο

| # | ΑΜ | GitHub profile | Approved |
| -- | -- | -- | -- |
| 1 | dpsd19085 | [ Profile](https://github.com/IoannaNi) | ok |
| 2 | dpsd17053 | [ Profile](https://github.com/kourtidaros) | ok |
| 3 | dpsd17127 | [Profile](https://github.com/SUPERSPECIAL99) | ok

**Reply by kourtidaros** [reply]
geiaaaa

**Reply by merkourisa** [reply]
@IoannaNi Delete the entry for the third member of your team?

**Comment by dpsd19126** [comment]
Name of the team: AS-KK 
GitHub url: [URL](https://github.com/dpsd19126/Interaction-Design-Project-Assignment)
Accepting new member:No

| # | ΑΜ | GitHub profile |Approved |
| -- | -- |-- |-- |
| 1 | dpsd19126 | [Profile](https://github.com/dpsd19126) | ok |
| 2 | dpsd19049 | [Profile](https://github.com/dpsd19049) | ok |
| 3 | dpsd19045 | [Profile]( https://github.com/Alexkarais)| ok |

**Comment by papadimitriou19093** [comment]
Name of the team: BRIANI
GitHub url: [URL](https://github.com/papadimitriou19093/Interaction-Design-Project-Assignment)
Accepting new member:No

|  #  | AM            |  Profile                                              | Approved | 
| --- |------------| --------------------------------------- | -- | 
|  1  |dpsd19093 |[THEODOSIS](https://github.com/papadimitriou19093)  | OK | 
|  2  |dpsd19104 |[IORDANIS](https://github.com/achtarmas) | OK | 
|  3  |dpsd19054 |[DIMITRIS](https://github.com/kokkinis19054) | OK |

**Reply by merkourisa** [reply]
@papadimitriou19093 please correct the GitHub URL. You have the URL of the individual assignment repository and not the coordinator's group assignment repository

**Comment by MarigiannaKotrotsiou** [comment]
Name of the team: Girlbosses
GitHub URL : [URL](https://github.com/MarigiannaKotrotsiou/Interaction-Design-Project-Assignment)
Accepting new member: No
| # |       AM         |    Names     | Approved | 
| -- | -------------  | ------------- | -- | 
| 1 | dpsd19057  |[Profile](https://github.com/MarigiannaKotrotsiou)| ok | 
| 2 | dpsd19073  | [Profile](https://github.com/XristinaMavropoulou)| ok | 
| 3 | dpsd17050  | [Profile](https://github.com/StellaKoronaiou17050) | ok |

**Comment by KostisG** [comment]
> Name of the team: RBK GitHub URL: [URL](https://github.com/KostisG/Interaction-Design-Project-Assignment/settings/pages) Accepting new member: Νο
> 
> 
> 

| #  | AM | Profile | Approved |
| ------------- | ------------- |  ------------- |  ------------- |
| 1 | dpsd19004  | [Profile](https://github.com/ReniaAnast)  |  ok |
| 2 | dpsd19019  | [Profile](https://github.com/dpsd19019)  |   ok |
| 3 | dpsd19025  | [Profile](https://github.com/KostisG)  |  ok |

**Reply by KostisG** [reply]
For some reason i cant make the board

**Reply by merkourisa** [reply]
@KostisG https://www.youtube.com/watch?v=3vXkgWtON9M

**Comment by dpsd14034** [comment]
Name of the team VaKa 
GitHub URL: [URL](https://github.com/dpsd14034/Interaction-Design-Project-Assignment) 
Accepting new member: No
 
| # | AM | PROFILE | APPROVED |
| --- | --- | --- | --- |
| 1 | dpsd14034 | [GiorgosK](https://github.com/dpsd14034) | OK |
| 2 | dpsd18013 | [dpsd18013](https://github.com/dpsd18013) | OK |

**Comment by dpsd19037** [comment]
Name of the team: Ilmavag
GitHub url: [URL](https://github.com/dpsd19082/Interaction-Design-Project-Assignment?fbclid=IwAR1pA_LyOUuq745-ycsBLD3cj_0TL9MMGQPZdMqouDHRgefnzvlUdnWIfIA) 
Accepting new member: Νο


|# | ΑΜ | Profile | Approved |
| :---         |     :---:      |     :---: | ---: |
| 1   |   dpsd19082  | [Malamati](https://github.com/dpsd19082)   | ok  |
| 2   | dpsd19037  | [Ilektra]( https://github.com/dpsd19037)     | ok  |
| 3  | dpsd19102 | [Vaggelis]( https://github.com/dpsd19102)     | ok  |

**Comment by AnastasiaKourfalidou** [comment]
Name of the team: Tsarloteam 
GitHub URL: [URL](https://github.com/AnastasiaKourfalidou/Interaction-Design-Project-Assignment)
Accepting new member: Yes

| # | ΑΜ | GitHub profile | Approved |
| -- | -- | -- | -- |
| 1 | dpsd19060| [ Profile](https://github.com/AnastasiaKourfalidou) | ok |
| 2 | dpsd19116 | [ Profile](https://github.com/dpsd19116) | ok |


_Originally posted by @AnastasiaKourfalidou in https://github.com/merkourisa/Interaction-Design-Project-Assignment/discussions/2#discussioncomment-2292977_

**Comment by desp5** [comment]
Name of the team: Gold Diggersss
GitHub url: [URL](https://github.com/desp5/Interaction-Design-Project-Assignment) 
Accepting new member: Νο


|# | ΑΜ | Profile | Approved |
| :---         |     :---:      |     :---: | ---: |
| 1   |   dpsd19012  | [desp](https://github.com/desp5)   | ok  |
| 2   | dpsd19056  | [giorgis](https://github.com/dpsd19056)     | ok  |
| 3  | dpsd19076 | [mike](https://github.com/mm0al)     | ok  |

_Originally posted by @dpsd19012 in https://github.com/merkourisa/Interaction-Design-Project-Assignment/discussions/2#discussioncomment-2387589_

**Reply by merkourisa** [reply]
@desp5 Which is the name of the team?

**Comment by dimitrisl11** [comment]
Name of the team: a little late to the party
GitHub [url](https://github.com/dimitrisl11/Interaction-Design-Project-Assignment)
Accepting new member: Νο


|# | ΑΜ | Profile | Approved |
| :---         |     :---:      |     :---: | ---: |
| 1   |   dpsd19067  | [Profile](https://github.com/dimitrisl11) | ok  |
| 2   | dpsd19068  | [Profile  ](https://github.com/Elenalyra)     | ok  |

**Comment by Dpsd19022** [comment]
Name of the team: Bucaneros
GitHub <a href = "https://github.com/NaiEimaiOAvraam/Interaction-Design-Project-Assignment"> URL</a> 
Accepting new member: Νο
| AM | Profile |  Approved |
| ------------- | ------------- | -- |
| dpsd19022  | <a href = "https://github.com/Dpsd19022"> Profile</a>  | OK |
| dpsd19002  | <a href = "https://github.com/NaiEimaiOAvraam"> Profile</a>   | OK  |
